/chassis:
  ros__parameters:
    imu:
      # mode zyz/atk/mpu6050/mpu9250
      module: "mpu6050"
      # IMU端口
      port: "/dev/i2c-0"
      # 波特率
      baudrate: 921600
      # 中断脚 PI16
      interupt:
        chip: "/dev/gpiochip0"
        line: 272
    remote:
      # 遥控类型 sbus/gamepad/keyboard/socket/sonnyps2
      type: "sbus"
      # 端口
      port: "/dev/ttyS5"
      # 波特率
      baudrate: 100000
      # 1帧数据长度
      data_len: 25
      # 最大量程
      joy_var_max: 1800
      # 最小量程
      joy_var_min: 199
      # 线速度最大值 单位m/s
      max_x_vel: 0.4
      # 角速度最大值 单位rad/s
      max_w_vel: 5.0
      # 最大转向角
      max_angle: 0.758
    driver:
      chip: "Two-wheel"
      bts7960_lis: -1
      bts7960_ris": -1
      moto1_ena: "moto1-ena"
      moto1_enb: "moto1-enb"
      moto2_ena: "moto2-ena"
      moto2_enb: "moto2-enb"
      lidar_pwm:
        chip: 0
        channel: 1
        polarity: true
        period: 50000000
        dutycycle: 20000000
      moto_pwm_left: 
        chip: 0
        channel: 3
        polarity: true
        period: 40000
        dutycycle: 0
      moto_pwm_right:
        chip: 0
        channel: 4
        polarity: true
        period: 40000
        dutycycle: 0
      RotaryEncoder: 
        - "rotary@1"
        - "rotary@0"
    car_param:
      LeftWheelRadius: 0.0315
      RightWheelRadius: 0.0315
      TrendLength: 0.15
      WheelBase: 0.1
      Ticks1Roll: 494.5
      MaxXVel: 0.389079
      MaxWVel: 5.18772
      MaxAngle: 0.758
      ZeroAngle: 0.0
      MaxXAcc: 0.5
      MaxWAcc: 0.5
      MaxAngerLimit : 0.785399
      MinAngerLimit : -0.785399
      # K 比例系数
      Proportion: 100.0
      # I 积分系数
      Integration: 1.25
      # D 微分系数
      Differentiation: 0.0
    # 超声波
    srf04: false
    # tof激光测距
    vl53l0x: false
    # 录制数据开关
    record_data: false
    # 视频设备
    video_device:
      # 直播开关
      live_video: false
      dev: "/dev/video1"
      format: "YUV"
      width: 640
      height: 480
      fps: 30
